# controller.yaml

joint_trajectory_controller:
  type: "effort_controllers/JointTrajectoryController"
  joints:
    - xPrismatic1
    - yPrismatic
    - zPrismatic
  gains:
    xPrismatic1: {p: 100.0, i: 0.01, d: 0.1}
    yPrismatic: {p: 100.0, i: 0.01, d: 0.1}
    zPrismatic: {p: 100.0, i: 0.01, d: 0.1}
  constraints:
    goal_time: 0.6
    stopped_velocity_tolerance: 0.01
  state_publish_rate: 50
  action_monitor_rate: 20
  hold_trajectory_duration: 0.0